bodhi/server/migrations/versions/abc52bce138c_added_field_to_store_email_template_.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2018 Red Hat, Inc.
+#
+# This file is part of Bodhi.
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+"""
+Added field to store email template filename for releases.
+
+Revision ID: abc52bce138c
+Revises: 39132c56a47e
+Create Date: 2018-07-05 15:04:37.212447
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'abc52bce138c'
+down_revision = '39132c56a47e'
+
+
+def upgrade():
+    """Add a template field to the releases table."""
+    op.add_column('releases', sa.Column('template', sa.UnicodeText(), nullable=True))
```

Comment:
You are creating the template column here but you do not set any value to it. It is also not used in the code in order to select the correct template for a release.

You should probably change the logic around [here](https://github.com/fedora-infra/bodhi/blob/develop/bodhi/server/models.py#L2793) to use the value stored in `template`.

Also I think `template` is a little bit too generic, let call it  `mail_template` so we know the purpose of this template :)

Reply:
Do we have to write SQLAlchemy ```UPDATE``` statements inside migration ```upgrade()```? Or is there another way we can write statements to update ```mail_template``` value of ```releases``` table?

Reply:
Yes you should use the `upgrade` function to add these statements. So when the new column is added it will be populated with the correct information too